### PR TITLE
Fix colors not showing

### PR DESCRIPTION
**Pull request by houfio**
Also fixes the `&nbsp;` on newlines.

See #196

### CODE REVIEW

File: test/fixtures/verbose/index.js [hunk 1]
```diff
@@ -1 +1 @@
-setInterval(() => console.log(Math.random()), 100)
+setInterval(() => console.log('[31m' + Math.random() + '\n'), 200)
```
Comment (j-f1) [inline test/fixtures/verbose/index.js]: Does this have an escape cgaracter in it? If so, can you put it in the string with a backslash escape so it’s easier to read?

File: src/app/formatter.ts [hunk 2]
```diff
@@ -12,7 +11,6 @@ export function formatLines(str: string): string[] {
     .split('\n')
     .map(blankLine)
     .map(ansi2HTML)
-    .map(escapeHTML)
```
Comment (j-f1) [inline src/app/formatter.ts]: Does this open Hotel up to XSS attacks?

File: test/fixtures/verbose/index.js (hunk 1 above)
Comment (houfio) [reply test/fixtures/verbose/index.js]: Sorry, I don't really get what you mean. Nothing in the string needs escaping.

**Comment by typicode** [comment]
Thank you!

**Pull request merged**